comm/index.html

```diff
@@ -9,64 +9,84 @@ <h1>Community support</h1>
 The GeoServer community utilizes multiple channels of communication. This page should help you find the channel best suited for your needs.
 </p>
 
-<a id="mail"></a>
-<section>
-    <h2>Mailing Lists</h2>
-    <p>
-    Email is the primary form of communication for GeoServer users and developers. Being an 
-    open community means that sending email to an individual developer is discouraged. Instead
-    all email communication should take place on the email lists. If you have a question,
-    chances are someone else has had the same question.
-    </p>
-    <p>
-    If you are not in position to communicate in public please contact one of our <a href="../support">commercial support</a> vendors.
-
     <a id="disclosure"></a>
     <section>
         <h3>Responsible Disclosure of Security Issues</h3>
     
         {% include disclosure.html %}
     </section>
 
+
+<a id="discourse"></a>
+    <h2>Discourse</h2>
+    <p>
+    The GeoServer-Users mailing list has migrated to <a href="https://discourse.osgeo.org/c/geoserver/users">Discourse</a>, an opensource web-based forum platform provided by OSGeo. All 20+ years of email archives are available to be searched and built upon, alongside other OSGeo greats such as QGIS, PostGIS, pgRouting, etc.</br>
```

Comment:
```suggestion
    The GeoServer User forum is hosted  on <a href="https://discourse.osgeo.org/c/geoserver/user">discourse</a> provided by the Open Source Geospatial Foundation. The forum contains all 20+ years of content from the archived Source Forge email list.</br>
```

Reply:
User or Users?

Discourse is capitalised: https://en.wikipedia.org/wiki/Discourse_(software)

SourceForge is one word

Reply:
I cannot tell if it is GeoServer Users or Users as it is in consistent.

* User interface says "GeoServer" "User"
* URL says geoserver-users
* Email address says geoserver-user@discourse.osgeo.org

I asked for help here: https://discourse.osgeo.org/t/testing-new-topic-by-mail/49596/7

But I think we best use "geoserver-user" to match the email address.